server/routes/index.js

```diff
@@ -45,26 +45,24 @@ router.route('/meta')
 router.route('/preview')
   .get(eventPreviewController.findAll)
 
+// Public API routes when festivalTicketPrice === 0
+router.use(checkUserStatus)
+
+router.use('/events', eventRoutes)
+router.use('/places', placeRoutes)
+
 // Private API routes
 router.use('/*', (req, res, next) => {
-  passport.authenticate('jwt', { session: false }, (err, user) => {
```

Comment:
I would bring that back but move it into an own middleware (see my comments below) and call it here via `router.use(authorizeJWT)` or similar